pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -868,6 +868,16 @@ func sortUnpackJobs(jobs []*batchv1.Job, maxRetainedJobs int) (latest *batchv1.J
 		if failedI != failedJ {
 			return !failedI // non-failed job goes first
 		}
+		// sort multiple jobs without the Failed condition
+		if condI == nil && condJ == nil {
+			return jobs[i].CreationTimestamp.After(jobs[j].CreationTimestamp.Time)
+		}
+		if condI == nil {
+			return false
+		}
+		if condJ == nil {
+			return true
+		}
```

Comment:
is there a test case we can add to this somehow?

Reply:
+1 
https://github.com/operator-framework/operator-lifecycle-manager/blob/889ae50a66fe36a46a3c756a0ad7b481b73672f1/pkg/controller/bundle/bundle_unpacker_test.go#L1977

Reply:
Test cases here should check for this now: https://github.com/ankitathomas/operator-lifecycle-manager/pull/1